dex/elrond_dex_farm/src/rewards.rs

```diff
@@ -1,41 +1,37 @@
 elrond_wasm::imports!();
 elrond_wasm::derive_imports!();
+use super::config;
 
 type Nonce = u64;
+type Epoch = u64;
+const DIVISION_SAFETY_CONSTANT: u64 = 1000000000000;
```

Comment:
i think it would be better to have it in storage - and do storageLoadAsBigInt.

Reply:
done